MV.Wildebeest.Core/source/core/java/co/mv/wb/plugin/base/BaseMigration.java

```diff
@@ -71,4 +71,11 @@ public UUID getMigrationId()
 	{
 		return this.toState;
 	}
+
+	@Override
+	public String toString() {
+		String fromState = this.fromState.isPresent() ? this.fromState.get() : "_";
```

Comment:
Use Optional.getOrElse() here.  E.g. String fromState = this.fromState.getOrElse("_")